tests/bluetooth/tester/src/btp/btp_aics.h

```diff
@@ -0,0 +1,25 @@
+/* btp_aics.h - Bluetooth tester headers */
+
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/sys/util.h>
+
+#define AICS_READ_SUPPORTED_COMMANDS	0x01
+
+#define AICS_SET_GAIN			0x02
+#define AICS_MUTE			0x03
+#define AICS_UNMUTE			0x04
+#define AICS_MAN_GAIN			0x05
+#define AICS_AUTO_GAIN			0x06
+#define AICS_MAN_GAIN_ONLY		0x07
+#define AICS_AUTO_GAIN_ONLY		0x08
+#define AICS_DESCRIPTION		0x09
+#define AICS_MUTE_DISABLE		0x0a
```

Comment:
General comment: In Zephyr we only use tabs to indent until the first non-tab character, then afterwards we use spaces, so this should be 

```suggestion
#define AICS_SET_GAIN                   0x02
#define AICS_MUTE                       0x03
#define AICS_UNMUTE                     0x04
#define AICS_MAN_GAIN                   0x05
#define AICS_AUTO_GAIN                  0x06
#define AICS_MAN_GAIN_ONLY              0x07
#define AICS_AUTO_GAIN_ONLY             0x08
#define AICS_DESCRIPTION                0x09
#define AICS_MUTE_DISABLE               0x0a
```

This applies multiple other places in this PR